SparkPost/2web2ui
# FE-840 Add HTTP response status and redux action to Sentry logs

### What Changed
 - Added tags: httpResponseStatus and reduxActionType to sentry reports on requests

### How To Test
 - Cause an error in the UI (Make up a template id and try to view it, etc.)
 - Look in sentry log for the event, check that the tags are there and are complete.
 - Ensure you can filter by that httpResponseStatus key in sentry
